workspaces/marketplace/packages/cli/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@red-hat-developer-hub/marketplace-cli",
+  "description": "CLI for the Marketplace",
+  "version": "0.0.1",
+  "backstage": {
+    "role": "cli"
+  },
+  "private": true,
```

Comment:
I think we can already release a 0.0.1. For this please remove the private attribute.

```suggestion
```